scheds/rust/scx_lavd/src/bpf/main.bpf.c

```diff
@@ -2951,6 +2966,10 @@ s32 BPF_STRUCT_OPS_SLEEPABLE(lavd_init)
 	if (err)
 		return err;
 
+	/*
+	 * Initilize the current logical clock.
+	 */
+	WRITE_ONCE(cur_logical_clk, now);
```

Comment:
I wonder whether it'd be less confusing to start `cur_logical_clk` from some set value (e.g. 0) rather than wallclock time.

Reply:
Thanks @htejun for the suggestion. I will update it and made some minior changes then merge it.